### PR TITLE
fix(admin-gui): english spelling

### PR DESCRIPTION
**Pull request by xkureck**

### CODE REVIEW

File: apps/admin-gui/src/assets/i18n/en.json [hunk 1]
```diff
@@ -1161,26 +1161,26 @@
...
-      "LABEL_DESCRIPTION": "Label displayed to users to identify item on application form. If empty, \"Short name\" from basic settings is used as fallback.",
+      "LABEL_DESCRIPTION": "Label displayed to users to identify the item on the application form. If empty, \"Short name\" from basic settings is used as a fallback.",
       "LABEL_BUTTON_DESCRIPTION": "Label displayed on submit button.",
       "HELP": "Help",
-      "HELP_DESCRIPTION": "Help text displayed to user along with input widget.",
+      "HELP_DESCRIPTION": "Help text displayed to a user along with input widget.",
       "ERROR": "Error",
-      "ERROR_DESCRIPTION": "Error message displayed to user when enters wrong value.",
+      "ERROR_DESCRIPTION": "Error message displayed to a user when enters a wrong value.",
       "REGEX": "Regular expression",
```
Comment (bodnara) [inline apps/admin-gui/src/assets/i18n/en.json:1170]: "...when wrong value is entered".

File: apps/admin-gui/src/assets/i18n/en.json [hunk 2]
```diff
@@ -1243,23 +1243,23 @@
       "EDIT_HINT": "You don't have permission to edit notifications.",
       "BASIC_SETTINGS": "Basic settings",
       "EMAIL_TYPE": "Email type",
-      "EMAIL_TYPE_DESCRIPTION_CREATE": "Selected type of notification (action which trigger sending and who is notified).",
-      "EMAIL_TYPE_DESCRIPTION_EDIT": "Type of notification (action which trigger sending and who is notified).",
+      "EMAIL_TYPE_DESCRIPTION_CREATE": "Selected type of notification (action which triggers sending and who is notified).",
+      "EMAIL_TYPE_DESCRIPTION_EDIT": "Type of notification (action which triggers sending and who is notified).",
       "APPLICATION_TYPE": "Application type",
```
Comment (bodnara) [inline apps/admin-gui/src/assets/i18n/en.json:1248]: Info in parentheses is unclear and in my opinion could be structured better.

File: apps/admin-gui/src/assets/i18n/en.json [hunk 3]
```diff
@@ -1394,17 +1394,17 @@
       "TITLE": "Change status",
       "CURRENT_STATUS": "Current status: ",
       "VALID_DESCRIPTION": "Member is properly configured and has access to all provided resources.",
-      "INVALID_DESCRIPTION": "Member have configuration error and DON'T have access on provided resources. You can check what is wrong by changing member's status to ACTIVE. If possible, procedure will configure all necessary settings by itself.",
+      "INVALID_DESCRIPTION": "Member has configuration error and DON'T has access to provided resources. You can check what is wrong by changing the member's status to ACTIVE. If possible, the procedure will configure all necessary settings by itself.",
       "EXPIRED_DESCRIPTION": "Member didn't extend membership and might NOT have access to all services.",
```
Comment (bodnara) [inline apps/admin-gui/src/assets/i18n/en.json:1398]: "don't have" or "doesn't have"

File: apps/admin-gui/src/assets/i18n/en.json [hunk 4]
```diff
@@ -1511,7 +1511,7 @@
     },
     "REMOVE_DESTINATIONS": {
       "TITLE": "Remove destination from service",
-      "WARNING": "Removing destination will stop propagation of service's configuration for this destination/service.",
+      "WARNING": "Removing the destination will stop a propagation of the service's configuration for this destination/service.",
       "DESCRIPTION": "Following destinations will be removed.",
```
Comment (bodnara) [inline apps/admin-gui/src/assets/i18n/en.json:1515]: I would say no article before propagation.

File: apps/admin-gui/src/assets/i18n/en.json [hunk 5]
```diff
@@ -1723,15 +1723,15 @@
       "CANCEL": "Cancel",
       "SEARCH": "Search",
       "EMPTY_SEARCH_MESSAGE": "Search field cannot be empty",
-      "SEARCH_HINT": "Type in user`s First name, Last name (or both) or Login or Email.",
-      "NO_IDENTITIES": "No user matching search criteria found.",
+      "SEARCH_HINT": "Type in user`s First name, Last name (or both), or Login or Email.",
+      "NO_IDENTITIES": "No user matching search criteria were found.",
       "SUCCESS": "Identity was connected."
```
Comment (bodnara) [inline apps/admin-gui/src/assets/i18n/en.json:1728]: Change ' ` ' to " ' ". 
user -> users.

File: apps/admin-gui/src/assets/i18n/en.json [hunk 6]
```diff
@@ -1996,15 +1996,15 @@
...
       "SUCCESS_PWD": "Password has been set successfully",
       "LOGIN_FORMAT": "Login has to start with a letter and consist only of characters: a-z0-9_-",
       "LOGIN_LONG": "Login is too long",
       "LOGIN_SHORT": "Login is too short",
       "LOGIN_EXISTS": "Login already exists",
       "NEXT": "Next",
       "BACK": "Back",
-      "PASSWORD_NOT_SUPPORTED": "Currently selected namespace does not support setting of the password.",
+      "PASSWORD_NOT_SUPPORTED": "The selected namespace does not support the setting of the password.",
       "EMAIL_INVALID": "Enter a valid email address",
```
Comment (bodnara) [inline apps/admin-gui/src/assets/i18n/en.json:2008]: No article before  setting

**Comment by bodnara** [comment]
Also scope should be `admin` i guess.

File: apps/admin-gui/src/assets/i18n/en.json (hunk 2 above)
Comment (xkureck) [reply apps/admin-gui/src/assets/i18n/en.json:1248]: unfortunately, this whole situation is unlucky with the use of the text in the context, 
image
I think this needs more like rework, and that's not part of the PR

**Review by bodnara — APPROVED** [review]

**Pull request merged**